docs/languages/javascript/get-country-list/index.md

```diff
@@ -2,6 +2,121 @@
 title: Get list of countries
 sidebar_label: Get list of countries
 sidebar_position: 2
+tags:
+  - country_list
+  - javascript
+keywords:
+  - country_list
+  - javascript
+description: How to get list of countries in deriv api?
 ---
 
-For setting up the websocket connection....
+:::caution
+You can learn more about countries [here](/docs/core-concepts/residence-list)
+:::
+
+To get countries list, please update the `open` event listener like so:
```

Comment:
change `countries list` to `the countries list`

Reply:
yeah cool, thanks :pray: